util/microservice-sdk/js/test/unit/kafka/Consumer.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/no-unresolved */
 // eslint-disable-next-line import/extensions
```

Comment:
I think It can be removed:
"/* eslint-disable import/no-unresolved */
// eslint-disable-next-line import/extensions"

Reply:
Ok. I'll remove them.